src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,21 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <>
+    {comments.length > 0 ? (
```

Comment:
Don't render this component if no comments. Move this checking before `CommentList` component render

Reply:
Done ✅